CONTRIBUTING.md

```diff
@@ -59,3 +91,10 @@ If your code intentionally changes some of the existing snapshots, simply blow a
 rm -rf test/output
 yarn test
 ```
+## Submitting a pull request
+Once you have completed your feature and confirmed that all test have passed using `yarn test`, you should push your changes up to GitHub and submit a [pull request](https://docs.github.com/en/pull-requests/collaborating-with-pull-requests/proposing-changes-to-your-work-with-pull-requests/creating-a-pull-request).
+
+```bash
+# Push changes up to a new branch on GitHub
+git push -u origin user-name/some-feature
```

Comment:
```suggestion
git push -u origin user-name/some-feature
```

I don't think this workflow is generally available. You should first fork the repo, then push to a new branch on your fork, and finally make a pull-request against the official repo.

Reply:
This is interesting -- I wonder what a good way to callout different processes for our colleagues might be...

Reply:
I’m not sure it’s appropriate to cover the basics of git, GitHub, terminal commands here. It might be better to link to the corresponding GitHub tutorials, e.g.,

https://docs.github.com/en/get-started/quickstart/fork-a-repo
https://docs.github.com/en/get-started/quickstart/contributing-to-projects